relayer/relays/execution/main.go

```diff
@@ -222,7 +222,7 @@ func (r *Relay) fetchUnprocessedParachainNonces(latest uint64) ([]uint64, error)
 	latestBucket := latest / 128
 
 	for b := uint64(0); b <= latestBucket; b++ {
-		encodedBucket, err := types.EncodeToBytes(types.NewU128(*big.NewInt(int64(b))))
+		encodedBucket, err := types.EncodeToBytes(types.NewU64(b))
```

Comment:
Is this related to the subxt change?

Reply:
Nope, it's to coordinate with https://github.com/paritytech/polkadot-sdk/pull/8240, which includes an optimized SparseBitmap.

Reply:
Good catch, thank you @yrong!